src/routes/backoffice.ts

```diff
@@ -67,6 +67,47 @@ export const triggerBookingsWebhook = async (solarisAccountId) => {
   await triggerWebhook(TransactionWebhookEvent.BOOKING, payload);
 };
 
+export const setAccountSeizureProtectionHandler = async (req, res) => {
```

Comment:
This method supports setting seizure protection from the panel. In end to end tests of the other projects, we set some values programatically, by sending requests to these endpoints. 
And exmaple of this is, the setIdentification and setIdentificationState functions. One is made for the mocksolaris panel and another is made to be called by e2e tests. The main difference is, one redirects so that the values are reloaded and the other returns http 200 so that e2e test knows the value is set. If you are creating e2e tests for this feature in other projects, you'll either need this functionality, or set seizure protection as you set the account data, and in that case you'll need to make sure seizure protection data is written to the account by the function which sets the accountdata.

Reply:
Oh I was looking at the e2e this morning which is why I changed the param to email but it completely slipped my mind that I also need to alter the returned values. Thank you!

Reply:
There are two ways, you can add a copy of the function as it is done on setIdentification and setIdentificationState or for one of them you can post one additional value to determine where the request is initiated from, and if panel, the function would redirect and if e2e, function would return 200

Reply:
yeah I saw this approached used earlier I think I'm gonna go for it
```
if (shouldReturnJSON(req)) {
    res.send(person);
  } else {
    res.render("person", {
      person,
      mobileNumber,
      taxIdentifications,
      devices,
      identifications: person.identifications,
      SEIZURE_STATUSES,
    });
  }